changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## NEXT
+
+- Made `cai_causal_graph.causal_graph.CausalGraph.copy` more general to preserve the class type. Removed `.copy` from
+  the `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph` class.
```

Comment:
Mention you extended the equality methods for the graphs. See 0.2.3 for language used prior.

Reply:
adding now.